metadata-ingestion/src/datahub/ingestion/source/looker/looker_source.py

```diff
@@ -286,7 +286,8 @@ def _get_views_from_fields(self, fields: List[str]) -> List[str]:
             except AssertionError:
                 self.reporter.report_warning(
                     title="Failed to Extract View Name from Field",
-                    message="The field was not prefixed by a view name. This can happen when the field references another dynamic field.",
+                    message="The field was not prefixed by a view name. This can happen when the field references "
```

Comment:
this reformatting stuff that your IDE is doing makes it extremely hard to review PRs. Please fix your setup

Reply:
it was pep8 max line warning in pycharm. I disabled it